srcpkgs/gamemode/template

```diff
@@ -25,17 +25,16 @@ post_install() {
 libgamemode_package() {
 	short_desc+=" - shared libraries"
 	pkg_install() {
-		# For compatibility reasons, all shlibs should go in libgamemode
-		vmove "usr/lib/*.so*"
+		vmove "usr/lib/*.so.*"
```

Comment:
`dlopen` now uses the versioned so `dlopen("libgamemode.so.0", RTLD_NOW);`, with a fallback to `libgamemode.so`